src/cloud-api-adaptor/entrypoint.sh

```diff
@@ -94,7 +94,7 @@ azure() {
 gcp() {
     test_vars GCP_CREDENTIALS GCP_PROJECT_ID GCP_ZONE PODVM_IMAGE_NAME
 
-    [[ "${PODVM_IMAGE_NAME}" ]] && optionals+="-gcp-image-name ${PODVM_IMAGE_NAME} "
+    [[ "${PODVM_IMAGE_NAME}" ]] && optionals+="-image-name ${PODVM_IMAGE_NAME} "
```

Comment:
I understand that this is just a transfer from gitlab, but still…

Frankly, unless we think there is any chance we can converge all the image IDs, I would go the other way. How can you possibly remember who is who between `imageid` and `image-id` for example??? Something like `aws-image` and `azure-image` is much easier on the eye and brain.

What about `aws-image`, `azure-image` and `gcp-image`?

Reply:
I don't disagree, but it's not really a transfer from gitlab. This code comes from upstream, it was cherry-picked to gitlab, and now I'm taking it here to be on the same level.
So if this code needs to be modified anywhere, it's upstream, not here.